### PR TITLE
Add redirects

### PR DESCRIPTION
**Pull request by ldemailly**
mostly as an example to add more as needed (like search)
cc @geeknoid

### CODE REVIEW

**Comment by geeknoid** [comment]
This doesn't seem right to me.

There is in fact a file 0.4/about/index.html. The link points to 0.4/about. It seems the problem is that the web server is not automatically converting a reference to  a directory to the reference to the index.html file inside that directory.

**Comment by ldemailly** [comment]
there is no directory there [in 0.3, 0.4]

the .html extension is missing in 0.3 and 0.4 (but not in 0.1 nor 0.2 nor 0.5)

File: archive.istio.io/firebase.json [hunk 1]
```diff
@@ -5,6 +5,16 @@
       "firebase.json",
       "**/.*",
       "**/node_modules/**"
+    ],
+    "redirects": [ {
+      "source" : "/v0.3/about",
```
Comment (ldemailly) [inline archive.istio.io/firebase.json]: @geeknoid it's `about` not `about/`

**Comment by ldemailly** [comment]
@geeknoid fyi I found a more generic solution:
https://github.com/istio/admin-sites/commit/9b3440f5dd21826357cc47bb65af94cf8dd8c3e8

this being said we should still probably have consistent use of directories / vs files foo.html vs files without the .html

**Comment by geeknoid** [comment]
So what's the semantic different between:

https://istio.io/about
https://istio.io/about/
https://istio.io/about/index.html

Aren't all three equivalent with a typical web server configuration?

**Comment by ldemailly** [comment]
no there is a big difference between `about` and `about/`
if in the html you have any other link, image or reference that are relative (which most links should be, instead of relying on a start from root ({{home}}) - they can't work for both

`<a href="foo/">` in `about` is `/foo` while in `about/` it's `about/foo/`

**Pull request merged**